### PR TITLE
healthcare: conditional samples

### PR DESCRIPTION
**Pull request by tbpg**
~Conditional patch is not working right now, but the sample looks correct
to me. So, I left out the region tag and test for it.~ Fixed.

cc @noerog

### CODE REVIEW

File: healthcare/fhir_test.go [hunk 1]
```diff
@@ -167,6 +170,44 @@ func TestFHIRStore(t *testing.T) {
 		}
 	})
 
+	testutil.Retry(t, 10, 2*time.Second, func(r *testutil.R) {
+		buf.Reset()
+		patchedRes := resource{}
+		if err := conditionalpatch.ConditionalPatchFHIRResource(buf, tc.ProjectID, location, datasetID, fhirStoreID, resourceType, false); err != nil {
+			r.Errorf("ConditionalPatchFHIRResource got err: %v", err)
```
Comment (jadekler) [inline healthcare/fhir_test.go]: return r.Errorf?

File: healthcare/internal/dicomweb-instance-search/dicomweb_instance_search.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	storesService := healthcareService.Projects.Locations.Datasets.DicomStores
+
+	parent := fmt.Sprintf("projects/%s/locations/%s/datasets/%s/dicomStores/%s", projectID, location, datasetID, dicomStoreID)
+
+	call := storesService.SearchForInstances(parent, dicomWebPath)
+	// Refine your search by appending DICOM tags to the
+	// request in the form of query parameters.
+	includeAllFields := queryParamOpt{key: "includefield", value: "all"}
+	resp, err := call.Do(includeAllFields)
+
```
Comment (jadekler) [inline healthcare/internal/dicomweb-instance-search/dicomweb_instance_search.go]: remove empty newline?

File: healthcare/internal/dicomweb-instance-search/dicomweb_instance_search.go [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	if err != nil {
+		return fmt.Errorf("SearchForInstances: %v", err)
+	}
+
+	defer resp.Body.Close()
+
+	if resp.StatusCode > 299 {
+		body, _ := ioutil.ReadAll(resp.Body)
```
Comment (jadekler) [inline healthcare/internal/dicomweb-instance-search/dicomweb_instance_search.go]: Why skip err check?

File: healthcare/internal/fhir-resource-conditional-patch/fhir_resource_conditional_patch.go [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+	call.Header().Set("Content-Type", "application/json-patch+json")
+
+	// Refine your search by appending tags to the request in the form of query
+	// parameters. This searches for resources updated in the last 48 hours.
+	twoDaysAgo := time.Now().Add(-48 * time.Hour).Format("2006-01-02")
+	lastUpdated := queryParamOpt{key: "_lastUpdated", value: "gt" + twoDaysAgo}
+
+	resp, err := call.Do(lastUpdated)
+
```
Comment (jadekler) [inline healthcare/internal/fhir-resource-conditional-patch/fhir_resource_conditional_patch.go]: remove newline

File: healthcare/internal/fhir-resource-conditional-update/fhir_resource_conditional_update.go [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+	call.Header().Set("Content-Type", "application/fhir+json")
+
+	// Refine your search by appending tags to the request in the form of query
+	// parameters. This searches for resources updated in the last 48 hours.
+	twoDaysAgo := time.Now().Add(-48 * time.Hour).Format("2006-01-02")
+	lastUpdated := queryParamOpt{key: "_lastUpdated", value: "gt" + twoDaysAgo}
+
+	resp, err := call.Do(lastUpdated)
+
```
Comment (jadekler) [inline healthcare/internal/fhir-resource-conditional-update/fhir_resource_conditional_update.go]: remove newline

File: healthcare/internal/dicomweb-instance-search/dicomweb_instance_search.go (hunk 3 above)
Comment (tbpg) [reply healthcare/internal/dicomweb-instance-search/dicomweb_instance_search.go]: Done.

File: healthcare/internal/fhir-resource-conditional-patch/fhir_resource_conditional_patch.go (hunk 4 above)
Comment (tbpg) [reply healthcare/internal/fhir-resource-conditional-patch/fhir_resource_conditional_patch.go]: Done.

**Pull request merged**